cpp/src/DataStorm/NodeSessionManager.h

```diff
@@ -77,7 +76,12 @@ namespace DataStormI
         // the specified node and sets this member once the connection is established.
         std::optional<DataStormContract::LookupPrx> _connectedTo;
 
+        // Stores the connection currently sending an announcement. This is used to exclude the node session
```

Comment:
Can it be null?

Reply:
It is always non null when forwarding calls in `NodeSessionManager::forward` which is the only place where it is read.